### ISSUE TITLE
To get the Halmak keyboard layout working on Fedora KDE Linux

### ISSUE THREAD

**Issue by Baigle** [MadRabbit/halmak#25]
I had to deviate from the instructions a little bit.

1. Copy the contents of /symbols/halmak and paste them into the /usr/share/X11/xkb/symbols/`us` file which contains all keyboard layout variants of standard US English, including Dvorak, Colemak, and Workman. Preferably locate it near where those other options are.
2. Add a line above `name[Group1]= "English (Halmak)";` that says `include "us"` in the symbols file.
3. Put the `halmak` variant near the Colemak or Dvorak list & description section in /rules/evdev.xml similar in formatting to the other variants. An example description is `English (Halmak)` or `AI superiority`.

Notes: 
- The < and > symbols, which are normally on the , and . keys, have been switched for the ( and ) symbols, making it far easier to program and type, but your keyboard keycaps may still show the wrong symbols.

**Issues**:
- The layout will switch all the key positions back to QWERTY when you hold down the CTRL key so that undo, cut, copy, paste, find, close window, select all, and other common muscle memory shortcuts work without remapping everything to new keys or reaching for awkward combinations. However, this does not always work as expected in every program, and I think it would just be better off without the hit-or-miss backwards compatibilities. _Disabling them in the qwerty-controls file_ seems to at least work with CTRL no longer switching keymaps, but there may still be a little bit of odd behavior depending on whether the program detects direct hardware input, or the translated inputs from xkb.

I find this far easier to use and learn versus ColemakDH or Qwerty. Thank you for sharing it with us, and telling us how to get it working on each of our operating systems. I tried for days to get it working on Clear Linux, Arch, FreeBSD, and Fedora before finding your repository.

**Comment by kaievns** [comment]
Yeah it's not perfect on linux. Basically all electron based apps and chrome/chromium are fucked. I couldn't figure how to sort those so i had to remap all those apps manually. Pain in the neck. I've seen people backing those mappings into a kernel patch bypassing the whole xkbd business, but that seems like madness.

So, i'm open for new ideas here.

As a side note, after years on halmak and ortho, i don't really care for QUERTY controls that much tbh. seems like it could go as well
